src/app/app.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'front';
+
+  constructor(private router: Router) { }
+
+  navigateForPage() {
+    this.router.navigate(['/onboarding/form']);
```

Comment:
why do we need this if we already have a route for onboarding feature module. Where is this fucntion called?